public/assets/sass/style.scss

```diff
@@ -65,6 +65,8 @@ $color3: rgb(38, 38, 38);
 
 html {
   background: $color1;
+  // REM UNIT //
+  font-size: 14px;
```

Comment:
remember I said something about the fact we need  to start using `rem` sizes for fonts?

so, I introduce, `rem`! :-) the `font-size` set here  on `html` is the basic font size for the entire site.

We should think about how to go about this. I would suggest not setting anything else based on font size for now (we don't know how it might effect the ui... it's safer to use `rem` for `font-size` and `line-height` and I think this should do it. everything without a certain height would change accordingly, and everything else (with a specific height), won't change.

But this change is not one of the most important ones, so we won't do a refactor for this soon. We still need to figure out our strategy on this.